cfn-templates/data-exports-aggregation.yaml

```diff
@@ -0,0 +1,749 @@
+            Action:
+              - s3:PutObject
+            Resource:
+              - Fn::Sub: 'arn:${AWS::Partition}:s3:::${DestinationS3}/*'
+          - Sid: AllowReplicationWrite
+            Effect: Allow
+            Principal:
+              AWS:
+                Fn::If:
+                  - EmptySourceAccountIds
+                  - !Ref AWS::AccountId
+                  - !Split [',', !Ref SourceAccountIds]
+            Action:
+              - s3:ReplicateDelete
+              - s3:ReplicateObject
+            Resource:
+              - Fn::Sub: 'arn:${AWS::Partition}:s3:::${DestinationS3}/*'
+          - Sid: AllowReplicationRead
+            Effect: Allow
+            Principal:
+              AWS:
+                Fn::If:
+                  - EmptySourceAccountIds
+                  - !Ref AWS::AccountId
+                  - !Split [',', !Ref SourceAccountIds]
+            Action:
+              - s3:ListBucket
+              - s3:ListBucketVersions
+              - s3:GetBucketVersioning
+              - s3:PutBucketVersioning
+            Resource:
+              - Fn::Sub: 'arn:${AWS::Partition}:s3:::${DestinationS3}'
+
+
+###########################################################################
+# Source Account Resources
+###########################################################################
+
+####
+# S3 Bucket with replication
+####
+
+  SourceS3:
+    Type: AWS::S3::Bucket
+    Condition: IsSourceAccount
+    DeletionPolicy: Delete
+    UpdateReplacePolicy: Delete
+    Properties:
+      BucketName:
+        Fn::Sub: ${ResourcePrefix}-${AWS::AccountId}-local
+
+      ## Uncomment following lines to enable bucket logging if needed. Please be careful with the cost of logging.
+      # LoggingConfiguration:
+      #   DestinationBucketName: REPLACE_WITH_YOUR_LOGGING_BUCKET
+      #   LogFilePrefix: REPLACE_WITH_YOUR_PREFIX
+
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256 ## Use AWS managed KMS
+              ## If you need Customer managed KMS key, yoy can do that using following parameters:
+              # SSEAlgorithm: aws:kms
+              # KMSMasterKeyID: "REPLACE_WITH_YOUR_KEY_ARN"
+      AccessControl: BucketOwnerFullControl
+      OwnershipControls:
+        Rules:
+          - ObjectOwnership: BucketOwnerEnforced
+      PublicAccessBlockConfiguration:
+        BlockPublicAcls: true
+        BlockPublicPolicy: true
+        IgnorePublicAcls: true
+        RestrictPublicBuckets: true
+      VersioningConfiguration:
+        Status: Enabled
+      ReplicationConfiguration:
+        Role: !GetAtt CrossRegionReplicationRole.Arn
+        Rules:
+          -
+            Destination:
+              Bucket:
+                Fn::Sub: "arn:${AWS::Partition}:s3:::${ResourcePrefix}-${DestinationAccountId}-shared"
+              StorageClass: STANDARD
+            Id: ReplicationRule1
+            Prefix: ""
```

Comment:
Need to manage this carefully. Please can you test creating athena table with 2 aggregated exports?

Reply:
+1 it should be leading to data folder to exclude metadata folder replication

Reply:
Should it be a filter on the ReplicationConfiguration? Or how else would I exclude metadata folder replication?

Reply:
you can try both. whatever works the best. I would suggest using the right prefix would be optimal as you know the path. Also this means that we need to use replication even if we have a local CUR (or alternatively prohibit cur to write */metadata/*).

Reply:
As discussed, we will have to use local replication, because: 1) S3 replication itself doesn't support exclusion 2) We can't exclude metadata folder in Athena 3) SCAD won't accept reduced bucket write permissions without metadata folder